ci/official/any.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+source "${BASH_SOURCE%/*}/utilities/setup.sh"
+
+# Parse options and build targets into arrays, so that shelllint doesn't yell
+# about readability. We can't pipe into 'read -ra' to create an array because
+# piped commands run in subshells, which can't store variables outside of the
+# subshell environment.
+# Ignore grep failures since we're using it for basic filtering
+set +e
+filtered_build_targets=( $(echo "$BUILD_TARGETS" | tr ' ' '\n' | grep . | tee build_targets.txt) )
```

Comment:
Probably missing something, why do we tee the targets out to a file and then check if that file exists rather than checking if there are entries in this `filtered_build_targets` array?

Reply:
Looks like build_targets.txt is no longer necessary after cl/471627157. Originally, the script checked build_targets.txt for PIP_WHEEL in some of the other conditionals, but now we can remove it.